fe/src/components/RankAndStats/RankAndStats.re

```diff
@@ -1,48 +1,33 @@
 open EloTypes;
-open Js.Promise;
 
-open Svc;
 [%bs.raw {|require('./RankAndStats.scss')|}];
 
-type state =
-  | LOADING
-  | LOADED(list(user))
-  | FAILURE;
+let component = ReasonReact.statelessComponent("RankAndStats");
 
-let component = ReasonReact.reducerComponent("RankAndStats");
+module GetUsers = [%graphql
```

Comment:
Nawiązując do pytania o ppx - to właśnie tutaj to jest użyte - dzięki tamtemu w bsconfig można tutaj zrobić takie zapytanie graphqlowe o takiej składni